apps/condo/domains/common/components/Comments/index.tsx

```diff
@@ -216,14 +188,18 @@ const CommentsTabPaneLabel = ({ label, commentsCount, newCommentsIndicator }) =>
         <Typography.Text style={COMMENTS_COUNT_STYLES}>
             {commentsCount}
         </Typography.Text>
-        {
-            newCommentsIndicator && (
-                <NewCommentIndicator title='' />
-            )
-        }
+        <NewCommentIndicator title=''/>
     </>
 )
 
+const SwitchCommentsTypeWrapper = styled.div`
```

Comment:
Aren't we moving away from `Antd` + `emotion` to `Condo UI` + `partial Antd` + `css modules`?